specs/toolsets/grpc_cppm/_build.ham

```diff
@@ -1,4 +1,4 @@
-if [ SubDirOnce TOP toolsets grpc_cppm ] = 1 { return ; }
+if [ SubDirOnce TOP specs toolsets grpc_cppm ] = 1 { return ; }
```

Comment:
Does that link up the grpc_cppm build correctly? How does it handle the fact that maybe some if its require toolsets are not present?

Reply:
grpc_cppm (and thrift_cppm) are not `SubInclude`ed in any other _build.ham files higher up the build tree, so the extra toolset requirements do not affect anything that is run from outside of this directory. Everything still links up and builds properly as well, for example if you manually delete your `ham/bin/(YOUR_LOA_HERE)/ham_thriftc_da` build artifact and then run `. hat thrift_cppm`, you will see the thrift_cppm toolset detect the missing artifact and rebuild it during the import same as before this change.